classes/core/PKPApplication.php

```diff
@@ -455,6 +462,11 @@ abstract public function getContextName(): string;
      */
     abstract public function getVersionDescriptorUrl(): string;
 
+    /**
+     * Get the review workflow stages used by this application.
+     */
+    abstract public function getReviewStages(): array;
```

Comment:
We can remove `abstract` and make this method returning an empty array by default as OPS doesn't have any.
Also we would need to override it on the application level (OJS and OMP).